tests/users_tests/test_users_views.py

```diff
@@ -70,3 +71,17 @@ def test_authorized_user_pages_access(self, client, url, template_file, new_user
         response = client.get(f'/{url}/')
         assert response.status_code == 200
         assertTemplateUsed(response, template_file)
+
+    def test_access_to_valid_user_profile(self, client, new_user):
```

Comment:
maybe add another test that shows which ResuMes are being displayed? (like how we did it on #131)

Reply:
Done!